ws/http/server.nim

```diff
@@ -15,6 +15,10 @@ import pkg/[
   chronicles,
   httputils]
 
+#required to log addresses in chronicle's json mode
+import json_serialization/std/net
```

Comment:
shouldn't we import json_serialization if only chronicles sink to json is defined?

Reply:
Yes, though not sure how to do this cleanly, eg
```nim
when config.streams[0].sinks[0].format == json:
  import json_serialization/std/net
```
This is horrible but works, we would need a helper function to check all sinks of all streams, and there doesn't seem to be one present in chronicles

Reply:
I think you only need to check for this `chronicles_sinks == "json"`, it is exported from chronicles.

https://github.com/status-im/nim-chronicles/blob/63ce43a86a40a4c73d1b3b8317278d47ec55a458/chronicles/options.nim#L14

Reply:
Oh, I thought they were other ways to create streams that would have a json sink, but sure i'll do that

Reply:
there can be multiple sinks however

Reply:
and there can be `chronicles_streams` that write json

Reply:
https://github.com/status-im/nim-chronicles/pull/99
When this is merge, we can just do
```nim
when config.isLogFormatUsed(json):
  import json_serialization/std/net```